apps/ui-kit/src/lib/components/organisms/dialog/Dialog.tsx

```diff
@@ -41,19 +41,24 @@ const DialogOverlay = React.forwardRef<
 >(({ showCloseIcon, position, ...props }, ref) => (
     <RadixDialog.Overlay
         ref={ref}
-        className={cx('dialog-overlay-bg inset-0 z-[99998] backdrop-blur-md', {
-            fixed: position === DialogPosition.Right,
-            absolute: position === DialogPosition.Center,
-        })}
+        className={cx(
```

Comment:
The dialog isn’t working the same as in production in the wallet, when the wallet is expanded, the overlay is covering the whole space, but it shouldn’t :see_no_evil: 
prod:
![image](https://github.com/user-attachments/assets/8c92c18a-6053-49ae-ae2e-9386a7929b0d)
current:
![image](https://github.com/user-attachments/assets/64e15635-35d7-4c2f-800e-3a205d92fca6)